SQL/0000-00-01-Permission.sql

```diff
@@ -106,7 +106,9 @@ INSERT INTO `permissions` VALUES
     (58,'electrophysiology_browser_view_allsites', 'View all-sites Electrophysiology Browser pages', 2),
     (59,'electrophysiology_browser_view_site', 'View own site Electrophysiology Browser pages', 2),
     (60,'battery_manager_view','View Battery Manager',2),
-    (61,'battery_manager_edit','Add, activate, and deactivate entries in Test Battery',2);
+    (61,'battery_manager_edit','Add, activate, and deactivate entries in Test Battery',2),
```

Comment:
RB_data needs to be added to run integration test. 
raisinbread/RB_files/RB_modules.sql 
INSERT INTO `modules` (`ID`, `Name`, `Active`) VALUES (40,'module_manager','Y');
raisinbread/RB_files/RB_permissions.sql
INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (62,'module_manager_view','Module Manager: access the module',2);
INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (63,'module_manager_edit','Module Manager: edit installed modules',2);